src/imgtools/utils/crawl.py

```
@@ -12,207 +13,198 @@
+                reference_ct,
+                reference_rs,
+                reference_pl,
+            ) = "", "", ""
+            try:  # RTSTRUCT
+                reference_ct = str(
+                    meta.ReferencedFrameOfReferenceSequence[0]
+                    .RTReferencedStudySequence[0]
+                    .RTReferencedSeriesSequence[0]
+                    .SeriesInstanceUID
+                )
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)